test/linalg/cholesky.jl

```diff
@@ -94,18 +91,12 @@ using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted, PosDefException
         U = Bidiagonal([2,sqrt(eltya(3))],[-1],true) / sqrt(eltya(2))
         @test full(chol(S)) ≈ full(U)
 
-        #lower Cholesky factor
```

Comment:
Hmm, I confused myself. Actually, there is no reason not to merge the two blocks, I will do that.